problem6.py

```diff
@@ -0,0 +1,18 @@
+#Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.
+
+#The list of numbers 1 to 100
+to_100 = list(range(1,100+1))
+
+#The sum of the squares of the first hundred natural numbers is:
+the_sum_of_squares = sum([next_squares**2 for next_squares in to_100])
+#print(the_sum_of_squares)
+
+#The square of the sum of the first ten natural numbers is:
+the_square_of_sum = sum([next_number for next_number in to_100])**2
+#print(the_square_of_sum)
+
+#the difference between the sum of the squares of the first hundred natural numbers and the square of the sum is:
+difference = the_square_of_sum - the_sum_of_squares
+print("The difference between the sum of the squares of the first one hundred natural numbers and the square of the sum is: %s " %difference)
+
```

Comment:
No main function and other functions

Reply:
you can create a function which takes the number of first natural number (in this problem it will be 100)